code/src/App.js

```diff
@@ -1,12 +1,36 @@
-import React from 'react'
-import data from './data.json'
+import React from 'react';
+import data from './data.json';
+import stretchGoal from './stretch-goal.json'; 
+import Footer from './Components/Footer.js';
+import Header from './Components/Header.js';
+import Playlist from './Components/Playlist.js';
+import Section from './Components/Section.js';
```

Comment:
There is a convention that says that we should divide outer dependencies (those located in node_modules) from local dependencies (all local files that we created - components, functions, data such as arrays) by one empty line. Moreover, style.css should be separated as well and located below local dependencies.